Controller/Component/LtiRequestComponent.php

```diff
@@ -168,7 +168,7 @@ public function authenticate() {
 		try {
 			$this->OAuthStore = ClassRegistry::init('Lti.OAuthStore');
 			$store = new OAuthStore($this->Provider, $this->Consumer);
-			$server = new OAuthServer($this->OAuthStore);
```

Comment:
I have no idea how this was working before, it should not have. Without this the consumer does not contain the secret so it cannot properly verify the signature.